### ISSUE TITLE
[TASK] SNAP data ETL prototype/MVP

### ISSUE THREAD

**Issue by Jarrus00** [WSU-CPTS415-ParquetParkour/Amazon-CoPurchasing#5]
_Background_:
Before we can perform queries and generate summaries, we will need a method to import data from the SNAP co-purchasing dataset to our chosen database (#2).  For the requirements of MS02, a Minimum Viable Product would consist of an end-to-end (extract, transform, and load) implementation for some portion of the dataset.

_Problem_:
We need a consistent means of setting up the database for downstream data processing.

_Success Criteria_:
1. Tasks #2 and #3 have been completed.
2. Data is able to be ingested from the SNAP dataset file into memory.
3. Any necessary transformations for loading into the database exist
4. Data may be inserted into the database.
5. Extract and transform actions preserve data associations/relations - parity exists between the SNAP dataset and the database.
6. _Stretch_: automate retrieval and decompression of dataset directly from SNAP to a predefined location in the project directory structure to keep the repo size down and further ensure consistency. (Should this be a separate task/improvement?)
